antsRegistration_affine_SyN.sh

```
@@ -793,96 +875,147 @@ function make_affine_pyramid {
       single_transform='Affine[ 0.1 ]'
     fi
 
+    local bins=32
+    if [[ "${linear_metric}" == "Mattes" && -n "${fixed_image:-}" ]]; then
+      IFS='x' read -r -a override_shrinks <<< "${linear_shrink_factors}"
+      local bin_arr
+      bin_arr=($(compute_mi_bins "${fixed_image}" "${override_shrinks[@]}"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)